waka_waka/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Godot 4+ specific ignores
+.godot/
```

Comment:
Go ahead and add the tmp file exclusion here (also @Space-Odin this is how you'll keep that obsidian settings file out of the repo)